jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/WaitConfiguration.java

```diff
@@ -120,9 +119,9 @@ private HttpConfiguration(String url, String method, String status) {
 
   public enum TcpConfigMode {
     // Use mapped ports
-    mapped,
+    MAPPED,
     // Use ports directly on the container
-    direct,
+    DIRECT,
```

Comment:
This will break possible deserialized configuration using lower-cased versions of these values

Reply:
Also, logic in line 87 of this same file will stop working too.

Please be very careful when changing enum fields. I'm more than happy to see enum field names with correct letter case, but right now we have a problem with how stuff gets deserialized from Maven configurations with plexus.